spec/suma/redis_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require "suma/redis"
+
+RSpec.describe Suma::Redis do
+  describe "#conn_params" do
+    it "returns keyword arguments" do
+      params = described_class.conn_params("redis://localhost:1234/0", reconnect_attempts: 1, timeout: 1.0)
+      puts params.inspect
+      expect(params[:url]).to eq("redis://localhost:1234/0")
```

Comment:
Same here, use `include`, not this enumeration of explicit keys. Also there is a `puts`.